models/maat/R/model2netcdf.MAAT.R

```diff
@@ -25,9 +25,6 @@ model2netcdf.MAAT <- function(outdir, sitelat = -999, sitelon = -999, start_date
   ## TODO is it OK to give site lat/long -999 if not running at a 'site'? 
   ## TODO !!UPDATE SO IT WILL WORK WITH NO MET AND WITH MET DRIVER!!
   
-  ### Load required libraries
-  library(PEcAn.utils)
```

Comment:
Some of the functions were already added properly (i.e. pkg::fcn) and I updated a few more then removed the need for library(PEcAn.utils)